tests/interfaces.ts

```diff
@@ -1,3 +1,5 @@
+import { DOCUMENT_STATES } from '@mia-platform-internal/fast-data-automation-lib/lib/interface/AggregationSchema'
```

Comment:
Shall we export the type from the `fast-data-automation-lib`, so that it belongs to the interface and it does not depends on where it is stored within the library itself?

Reply:
A quick `import { DOCUMENT_STATES } from '@mia-platform-internal/fast-data-automation-lib` won't work since the `DOCUMENT_STATES` is not included in the main lib export.

So that would be a change to be done in the Fast Data Automation Lib and, when this will be released, we can update the import accordingly.

Otherwise I'd prefer to not import at all the `DOCUMENT_STATES`, but define here a compatible type and wait for a better export in a future release.